src/analytics/formTracker/formTracker.js

```diff
@@ -0,0 +1,5 @@
+function formTracker() {
```

Comment:
If we want to use typescript, maybe it's better to use the extension .ts instead of js?

Reply:
Have switched file extensions in favour of typescript 👍